src/ucp/wireup/select.c

```diff
@@ -1062,7 +1062,7 @@ ucp_wireup_add_bw_lanes(const ucp_wireup_select_params_t *select_params,
         }
     }
 
-    return UCS_OK;
+    return num_lanes ? UCS_OK : UCS_ERR_UNREACHABLE;
```

Comment:
why?

Reply:
this is needed to do loop inside `ucp_wireup_add_rma_bw_lanes`

Reply:
maybe return num_lanes from this function instead of status?